etc/config/jakt.defaults.properties

```diff
@@ -0,0 +1,17 @@
+# Default settings for Jakt
```

Comment:
In order for the language to show up in the site, you'll need to create a `jakt.amazon.properties` file, which should look pretty much like this one.

This file is intended for local users to be able to run the site, without having to think too much about configuring it. That's why we usually relay on the PATH to find the `exe`s defined in the `.defaults` file.

Reply:
Just to be clear, do I still need the `.defaults` file at all? Because it's very unlikely anyone would have jakt installed in their PATH anywhere, same for the runtime. So I'm not sure what to put there. For now, I just put some defaults that might work in the future. I guess it's not too important, right?

Reply:
I don't know how the system would behave with a missing defaults file, so your solution seems like the better one to me, even though as you say, it's really not too important :)